pkgs/by-name/sp/spacectl/package.nix

```diff
@@ -2,6 +2,7 @@
   lib,
```

Comment:
```suggestion
  lib,
  stdenv,
```

Reply:
oops, sorry I should've left this as draft while I worked through these things